specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -482,79 +488,79 @@
       "type": "object",
       "properties": {
         "memoryInGB": {
-          "description": "The memory limit in GB of this container.",
+          "description": "The memory limit in GB of this container instance.",
           "type": "number",
           "format": "double"
         },
         "cpu": {
-          "description": "The CPU limit of this container.",
+          "description": "The CPU limit of this container instance.",
           "type": "number",
           "format": "double"
         }
       }
     },
     "AzureFileVolume": {
-      "description": "The Azure file volume.",
+      "description": "The properties of the Azure File volume.",
       "type": "object",
       "required": [
         "shareName",
         "storageAccountName"
       ],
       "properties": {
         "shareName": {
-          "description": "The Azure file share name.",
```

Comment:
We should decide on whether we want to call it "volume" or "share" AFAIK its interchangeable?